### DISCUSSION TITLE
Tagging repos containing stock assessments

### DISCUSSION THREAD

**Discussion by Schiano-NOAA** [nmfs-ost/asar#87] [Ideas]
Throwing the idea out there to see if we can start a movement to tag repositories since 2021 as stock assessments or adopt a tagging system so that these repositories are easier to find.

**Comment by iantaylor-NOAA** [comment]
Here are some examples of the "stockassessment" topic in use https://github.com/search?q=topic%3Astockassessment&type=repositories.
